pkg/lib/localstore/objectstore/queries.go

```diff
@@ -108,6 +108,22 @@ func (s *dsObjectStore) makeFTSQuery(text string, filters *database.Filters) (*d
 	return filters, nil
 }
 
+func getSpaceIDFromFilter(fltr filter.Filter) (spaceID string) {
+	switch f := fltr.(type) {
+	case filter.Eq:
+		if f.Key == bundle.RelationKeySpaceId.String() {
+			return f.Value.GetStringValue()
+		}
+	case filter.AndFilters:
+		for _, af := range f {
```

Comment:
May be 118-122 extract to separate function to reduce nested blocks

Reply:
Reduced nested blocks by introducing separate func for AndFilters iteration